client/src/App.css

```diff
@@ -1,3 +1,34 @@
-.App {
+body {
+  box-sizing: border-box;
+  background-color: #000;
+}
+
+.App,
+.App-header {
```

Comment:
Hi @jonnywyatt 
Thank you so much for your feedback.
Regarding the server.js file and my pull request, you're absolutely right. I intended to create branches for my levels to be more organized, but I ended up not having permission to merge it into my main branch. I got mixed up with a lot of things, probably due to the limitations set by the organization to prevent issues.

Additionally, with regards to the styles, I had my own styling initially, but when I discovered Material UI, I attempted to update all the components to use it.

Reply:
By the end of today,
I will make an effort to update and remove any unnecessary stuff. Please feel free to point out anything you've noticed.
Thank you very much!